### PR TITLE
add hironx_calibration

### PR DESCRIPTION
**Pull request by k-okada**
add hironx calibration package, this depends on https://github.com/ros-perception/calibration (need to merge https://github.com/ros-perception/calibration/pull/31)

Before calibrate (with no offset between head and camera)
hiro_before_calibrate

and after calibrate. It is not accurent enough but it may be due to using xition(kermine)...
hiro_after_calibrate

### CODE REVIEW

**Comment by k-okada** [comment]
tutorials -> http://wiki.ros.org/rtmros_nextage/Tutorials/CalibrateKinect

**Comment by k-okada** [comment]
Image to shows how to put xtion on head and put calibration board
hiro_xtion_calibborad

File: hironx_calibration/package.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  <maintainer email="kei.okada@gmail.com">Calibration Setup Helper</maintainer>
+
+  <license>BSD</license>
+  <url>http://ros.org/wiki/hironx_calibration</url>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <run_depend>calibration_launch</run_depend>
+  <run_depend>calibration_estimation</run_depend>
+  <run_depend>kdl</run_depend>
```
Comment (130s) [inline hironx_calibration/package.xml:20]: You might want to add `run_depend` on `calibration_setup_helper` here based on https://github.com/ros-perception/calibration/pull/31?

**Comment by 130s** [comment]
+1!

travis already passed before https://github.com/ros-perception/calibration/pull/31 gets merged maybe because it's only supposed to be `run_depend`ed. If that's correct then maybe we can merge this now.

**Comment by k-okada** [comment]
you can merge this,

we need calibration_setup_helper to create/generate this package, not to
run this package.

**Comment by garaemon** [comment]
Please use pr2_navigation_filter to filter out the points belong to robot
itself
https://github.com/PR2/pr2_navigation/pull/12#issuecomment-57047726

After hand-eye calibration, it should work

2014年9月27日土曜日、Kei Okadanotifications@github.comさんは書きました:

> you can merge this,
> 
> we need calibration_setup_helper to create/generate this package, not to
> run this package.
> 
> On Sat, Sep 27, 2014 at 1:11 AM, Isaac I.Y. Saito <
> notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>>
> wrote:
> 
> > +1!
> > 
> > travis already passed before ros-perception/calibration#31
> > https://github.com/ros-perception/calibration/pull/31 gets merged
> > maybe
> > because it's only supposed to be run_depended. If that's correct then
> > maybe we can merge this now.
> > 
> > —
> > Reply to this email directly or view it on GitHub
> > <
> > https://github.com/start-jsk/rtmros_hironx/pull/251#issuecomment-56983068>
> > 
> > .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/start-jsk/rtmros_hironx/pull/251#issuecomment-57038897
> .

## 

from iPhone

**Comment by 130s** [comment]
> @k-okada we need calibration_setup_helper to create/generate this package, not to run this package.

Got it. 

> @garaemon Please use pr2_navigation_filter to filter out the points belong to robot itself

Can you point to an example package that does this if anything?
And, that will be taken care of in another PReq. Merging this.

**Comment by garaemon** [comment]
You can get sample codes around [here](https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_pcl_ros/launch/organized_multi_plane_segmentation.launch#L35)

**Comment by 130s** [comment]
Pictures from this ticket are added to the tutorial. http://wiki.ros.org/rtmros_nextage/Tutorials/CalibrateKinect

**Comment by 130s** [comment]
Thanks!

> You can get sample codes around [here](https://github.com/jsk-ros-pkg/jsk_recognition/blob/master/jsk_pcl_ros/launch/organized_multi_plane_segmentation.launch#L35)

**Pull request merged**